paper-ripple.html

```diff
@@ -116,362 +113,418 @@
     }
 
   </style>
-
-  <div id="bg"></div>
-  <div id="waves">
-  </div>
-
-</template>
+  <template>
+    <div id="background"></div>
+    <div id="waves"></div>
+  </template>
+</dom-module>
 <script>
+  (function () {
 
-  (function() {
-
-    var waveMaxRadius = 150;
-    //
-    // INK EQUATIONS
-    //
-    function waveRadiusFn(touchDownMs, touchUpMs, anim) {
-      // Convert from ms to s
-      var touchDown = touchDownMs / 1000;
-      var touchUp = touchUpMs / 1000;
-      var totalElapsed = touchDown + touchUp;
-      var ww = anim.width, hh = anim.height;
-      // use diagonal size of container to avoid floating point math sadness
-      var waveRadius = Math.min(Math.sqrt(ww * ww + hh * hh), waveMaxRadius) * 1.1 + 5;
-      var duration = 1.1 - .2 * (waveRadius / waveMaxRadius);
-      var tt = (totalElapsed / duration);
-
-      var size = waveRadius * (1 - Math.pow(80, -tt));
-      return Math.abs(size);
-    }
+    var Utility = {
+      cssColorWithAlpha: function (cssColor, alpha) {
```

Comment:
This is not in the style guide but I think nowhere else in Polymer we use a space between `function` and the arguments.